scripts/e2e.sh

```
@@ -12,8 +12,7 @@ REDIS_CONTAINER="sippy-e2e-test-redis"
 REDIS_PORT="23479"
 
 if [[ -z "$GCS_SA_JSON_PATH" ]]; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)